scopes/design/ui/pages/standalone-not-found-page/standalone-not-found.compositions.tsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import { StandaloneNotFoundPage } from './index';
+
+export const StandaloneNotFoundExample = () => <StandaloneNotFoundPage />;
```

Comment:
Please check that you don't need to use ThemeContext in order to make the right styling.
```
import { ThemeContext } from '@teambit/documenter.theme.theme-context';
```

Reply:
Well, from the checks I did it all looked fine on the UI without it

Reply:
Looks fine without the ThemeContext